plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -18,6 +18,7 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
+import io.airlift.log.Logger;
```

Comment:
Could you run stress test in this PR?

Reply:
how to do stresstest?

Reply:
You can refer to https://github.com/trinodb/trino/pull/13625

Reply:
testing it now